test/unit/core/core.transformations/transformations.initialize.js

```diff
@@ -16,7 +16,7 @@ describe('Core Transformations', function() {
           }
         };
 
-        transformer = new Transformer(attributes, { models: {}, collections: {} });
```

Comment:
What does mean "model and collection properties"?
"model" and "collection" is just a object name? (same `{article: {...}, category: {...}}`)
I thought this test code means nested object like `{collections: {article: {...}, category: {...}}, models: {...}}`. So I fixed it.

Transformer's 2nd argument(table) is not used. So I cannot understand this argument.
But transformer initialized at `core/index.js` with `this.waterline.schema`. `this.waterline.schema` is simple object like `{article: {...}, category: {...}`. Maybe Transformer's 2nd argument doesn't expect nested object.